src/Theme.ts

```diff
@@ -1,15 +1,18 @@
 import { darken, invert, lighten, modularScale } from 'polished';
 
 const backgroundLight = 'white';
-const backgroundDark = '#21252b';
-export const borderRadius = '0.125em';
+const backgroundDark = '#214772';
+export const borderRadius = '6px';
```

Comment:
Do we want to increase the border radius for both light and dark themes? I believe this is 3 times larger, though I don't mind it.

Reply:
Resolved below